backend/api/v1/project_service.go

```diff
@@ -149,6 +150,22 @@ func getListProjectFilter(filter string) (*store.ListResourceFilter, error) {
 			positionalArgs = append(positionalArgs, v1pb.State(v1State) == v1pb.State_DELETED)
 			return fmt.Sprintf("project.deleted = $%d", len(positionalArgs)), nil
 		default:
+			// Check if it's a label filter (e.g., "labels.environment" == "production")
+			varStr, ok := variable.(string)
```

Comment:
Need to update the comment for the `filter` field in the `project_service.proto`